README.md

```diff
@@ -146,8 +146,18 @@ It is configured by subclassing and refining `cacheKey` method. A good key here,
 likely the name of the plugin, its version and the actual versions of its dependencies.
 
 ```js
-Subclass.prototype.cacheKey = function() {
- return md5(Filter.prototype.call(this) + inputOptionsChecksum + dependencyVersionChecksum);
+const Filter = require('broccoli-persistent-filter');
+
+class Subclass extends Filter {
+  constructor(inputNode, search, replace, options = {}) {
+    super(inputNode, {
+      annotation: options.annotation
+    });
+  }
+
```

Comment:
I think we don't need this constructor for the purposes of the example (which is only about `cacheKey`)

```suggestion
```